onnxruntime/wasm/api.cc

```diff
@@ -337,6 +338,12 @@ OrtRunOptions* OrtCreateRunOptions(size_t log_severity_level,
     RETURN_NULLPTR_IF_ERROR(RunOptionsUnsetTerminate, run_options);
   }
 
+  if (synchronize) {
```

Comment:
I see no difference in the if and else case.

Reply:
You are absolutely right ... I think i forgot to add to the changelist.